front/src/dayview/Dayview.module.scss

```diff
@@ -105,64 +107,76 @@ h2.headerText{
 
 .dayviewContainer .track {
   position: relative;
-  background:#b0aca0;
-  border:solid 1px black;
+  background: $COLOR_CREAM30;
+  border:solid 1px $COLOR_BLACK;
   box-sizing: border-box;
   min-height: 100px;
 }
 
 .dayviewContainer .clickableBox {
   box-sizing: border-box;
-  background:white;
+  background:$COLOR_WHITE;
   padding:25% 0vh 25% 0vh;
   text-align: center;
 }
 
 .dayviewContainer .greenB {
-  background:#658f60;
+  background: $COLOR_GREEN;
+}
+
+.infoItem .greenB {
+    background: $COLOR_GREEN;
 }
 
 .dayviewContainer .lightGreenB {
-  background:#b2d9ad;
+  background:$COLOR_GREEN_LIGHT;
 }
 
 .dayviewContainer .blueB {
-  background:#95d5db;
+  background: $COLOR_TURQUOISE;
 }
 
 .dayviewContainer .redB {
-  background:#c97b7b;
+  background:$COLOR_RED_LIGHT;
+}
+
+.infoItem .redB {
+    background:$COLOR_RED_LIGHT;
 }
 
 .dayviewContainer .whiteB {
-  background:#f2f0eb;
+  background:$COLOR_CREAM5;
+}
+
+.infoItem .whiteB {
+    background:$COLOR_CREAM5;
 }
 
 .dayviewContainer .yellowB {
-  background:#f2c66d;
+  background:$COLOR_ORANGE;
 }
 
-.dayviewContainer .arrow-right {
+.dayviewContainer .arrowRight {
   padding:0;
   margin:0;
   width: 0; 
   height: 0; 
   border-top: 20px solid transparent;
   border-bottom: 20px solid transparent;
-  border-left: 20px solid black;
+  border-left: 20px solid $COLOR_BLACK;
 
   display: flex;
   align-items: center;
 }
 
-.dayviewContainer .arrow-left {
+.dayviewContainer .arrowLeft {
```

Comment:
use @extend .arrow; in the arrows here and delete the overlapping style definitions

Reply:
Changed this one locally, will be on the next commit